shells/dev-shell/output-pane.js

```diff
@@ -223,7 +223,7 @@ const errorTemplate = `
     }
   </style>
   <div class="container">
-    <div id="header"></div>
+    <pre id="header"></pre>
```

Comment:
No need to change this; the `<pre id="message">` element should show the recipe.

Reply:
Neat, changed.